gitnamehere/point-of-sale-group-project
# Theming

This PR adds theming (only changes background color, header colors at this point) to the POS and some other stuff

Testing:
* `npm run setup-demo:dev`
* `npm run start:dev`
* login to the POS
* go to the settings page and scroll down to Themes
* play around with the colors and update it
* go to the home page, and verify the colors are updated
